scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Actually, let me clarify.  I was referring to an npm module for my changelog api, are you thinking of a release script for npm?